app/static/assets/robots.txt

```diff
@@ -0,0 +1,6 @@
+# All robots are allowed
+User-agent: *
+Disallow:
+
+# Sitemap files
+#Sitemap: ./sitemap.xml
```

Comment:
Just checking, did you mean to comment this line out? I looked at some sample robots.txt files and they don't do that...

Also, do we need to use the absolute URL, or is the relative URL also OK?

Reply:
I just checked and adjusted it. It was commented out when I got the file from Asma, but I just uncommented it.